<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ambulanta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="doktor_registruj_reg">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>160</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registruj se</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sifra_doktor_reg">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>189</y>
      <width>331</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="ime_doktor_reg">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>97</y>
      <width>331</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="doktorLoginSifra">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>146</y>
      <width>72</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Šifra:</string>
    </property>
   </widget>
   <widget class="QLabel" name="doktorLoginIme">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>54</y>
      <width>206</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Korisničko Ime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="doktorLoginLabel">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>10</y>
      <width>90</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doktor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_doktor_reg">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>281</y>
      <width>331</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>238</y>
      <width>42</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="greska_doktor_reg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
